### PR TITLE
display and add subjects using json objects

### PR DESCRIPTION
**Pull request by VigneshKarthigeyan**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -69,9 +82,21 @@ class TeamTomato(Resource):
     def get(self):
         return "Team Tomato welcomes you"
 
+class List(Resource):
+    def get(self):
+        return jsonify(questionPapers)
+
+class addToList(Resource):
```
Comment (AkbarHabeeb) [inline app.py]: Don't we need a capital letter here

**Comment by AkbarHabeeb** [comment]
Good attempt @VigneshKarthigeyan 

- Don't the class name begins with caps?
- Since its RESTful, we don't need to jsonify the results
- Post method invocation was okay, but it should not append it to the value directly in the program.
 There should be a DB call to do so.
- No hardcoding

File: app.py (hunk 1 above)
Comment (VigneshKarthigeyan) [reply app.py]: yeah bro
I forgot it

**Comment by AkbarHabeeb** [comment]
Its wrong @VigneshKarthigeyan 
Since you are using post and update, I wonder I am not able to find the database conncetion or any db related ones. Please redo.

**Comment by AkbarHabeeb** [comment]
Good job @VigneshKarthigeyan . Nice one.
Let this PR remain here, in the mean time convert all the db connection to use ORM and send a PR for it.

**Comment by AkbarHabeeb** [comment]
@VigneshKarthigeyan Looking good. Nice one.

1. Remove all the List related works in the script. Our aim is to create an API that supports the question paper site.
2. Create an API for adding the files to our DB
3. Create an API to update the data in our DB
4. Create an API to delete the data in our DB
5. Write everything to support the sqlAlchemy (ORM)

Note:
No need to write unwanted scripts here, kindly remove them

**Pull request closed without merging**